API/controller.js

```diff
@@ -79,3 +79,14 @@ exports.getUsers = (req, res, next) => {
     })
 }
 
+exports.deleteComment = (req, res, next) => {
+    const {comment_id} = req.params
+    deletingUsers(comment_id)
+    .then((comment) => {
+        res.status(204).send({msg: 'comment deleted'})
```

Comment:
nothing needs to be sent back here you can either leave off the .send or leave it empty :)